### DISCUSSION TITLE
ETA on Apt Repository for Ubuntu 24.04?

### DISCUSSION THREAD

**Discussion by pauljamesharper** [regolith-linux/regolith-desktop#1006] [General]
Is there an ETA on the Apt Repository for Ubuntu 24.04? This will be the new Ubuntu LTS (Long Term Support) version.

**Comment by kgilmer** [comment]
Hi Paul.  Ubuntu 24.04 support is in active development.  I have been using it in development for several weeks now.  There seem to be some late breaking changes that are impacting initialization of the X11 session with the released version of Noble.  Will update the thread when a beta is available for testing, hopefully quite soon.

**Reply by khos2ow** [reply]
Is there a PR opened for the fix you have in mind? Or in other words what thing should I start looking at to fix my login loop issue after upgrading to 24.04?

**Reply by kgilmer** [reply]
Hi @khos2ow , sure, [this PR ](https://github.com/regolith-linux/regolith-session/pull/32) represents the active troubleshooting going on with Noble.  The branch is currently [building `regolith-session-*` into `unstable`](https://github.com/regolith-linux/voulage/blob/main/stage/unstable/package-model.json#L9).  You're welcome to join in!  Regarding the issue you reported yesterday on the Wayland session, you may want to verify the apt line you used when installing.  (I was able to install and reach the Sway desktop in my 24.04 test environment)

**Reply by khos2ow** [reply]
Thanks, I'll take a look. Can you clarify what do you mean by this:

> you may want to verify the apt line you used when installing

`apt update && apt dist-upgrade` you mean?

**Reply by kgilmer** [reply]
Have a look here: https://github.com/regolith-linux/regolith-session/pull/32#issuecomment-2079539643

**Reply by khos2ow** [reply]
Great work! Switching apt source to unstable fixed the issue. I reached Sway desktop basically. I'll keep playing with it to see if I can find more issues.

**Comment by kgilmer** [comment]
For those wanting to test, the `Sway` session should work out of the box, but until a first beta release of Regolith 3.2 is available, the `i3` session is only usable if you also install `regolith-lightdm-config`.  This will cause your display manager to change, so again, only recommended for testing at this point.  This `apt` line will install `unstable`:

```shell
$ cat /etc/apt/sources.list.d/regolith.list 
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/unstable-ubuntu-noble-amd64 noble main
```

Bug reports welcome :smile:

**Comment by lukaspetersson** [comment]
Any updates on ETA @kgilmer ?

Love your work, regolith is great!

**Reply by kgilmer** [reply]
Hi @lukaspetersson ,  (thanks!) the current status is that Regolith 3.2 has a Beta 1 release.  

You can read the in-play release notes here:  https://regolith-desktop.com/docs/reference/Releases/regolith-3.2-release-notes/
We provide these updates via the [mailing list](https://www.freelists.org/list/regolith-linux), [Matrix](https://matrix.to/#/!UpUrlcKuTZQAXLMipN:matrix.org?via=matrix.org&via=mozilla.org&via=omg.lol), and [Reddit](https://www.reddit.com/r/regolithlinux/).

**Comment by ldesousa** [comment]
I have used Regolith on Ubuntu 24.04 the last two days. Mostly working fine, apart from the following:
* The bar does not show the programme icons (e.g. Nextcloud, Diodon, Telegram, etc)
* No longer possible to change icon size in the Settings (bug or feature?)
* The bar does not show battery status
